template.hbs

```diff
@@ -309,85 +344,73 @@
     {{#each Certificate.Inspection}}
     {{!-- Inspection-> SupplementaryInformation or ChemicalComposition or TensileTest,HardnessTest,NotchedBarImpactTest,OtherMechanicalTests --}}
     {{#typeOf this 'object'}}
-    <section class="second-level">
-      <h3 class="second-level-title second-level" style="float:left;">
+    <section>
+      <h3 class="second-level-title">
         {{ i18n @key "otherFields" ../Certificate.CertificateLanguages }}</h3>
 
       {{!-- Inspection-> SupplementaryInformation --}}
       {{#ifEqual @key "SupplementaryInformation"}}
       <table class="main-table">
         {{#each this}}
         <tr class="table-rows">
-          <th class="table-headers">{{@key}} {{this.Key}}</th>
+          <th class="table-headers" colspan="2">{{@key}} {{this.Key}}</th>
           <td class="table-cells">{{this.Value}} {{this.Unit}}</td>
           <td class="table-cells">{{this.Interpretation}}</td>
         </tr>
         {{/each}}
       </table>
       {{/ifEqual}}
-
-      {{!-- Inspection-> ChemicalComposition --}}
-      {{#ifEqual @key "ChemicalComposition"}}
-      {{#each this}}
-      {{#typeOf this 'object'}}
-      {{!-- ChemicalComposition-> SupplementaryInformation --}}
-      {{#ifEqual @key "SupplementaryInformation"}}
-      <h3 class="third-level-title second-level" style="float: left;">
-        {{ i18n @key "otherFields" ../../Certificate.CertificateLanguages }}</h3>
-      <table class="main-table second-level">
-        {{#each this}}
+     {{#ifEqual @key "ChemicalComposition"}}
```

Comment:
Can you add the comment `{{!-- Inspection-> ChemicalComposition --}}` ?
It's helping for the readability.